### DISCUSSION TITLE
MUI CheckBox doesn't work with current version

### DISCUSSION THREAD

**Discussion by bkorumaz** [react-hook-form/react-hook-form#3267] [Q&A] [answered]
Material-UI's Checkbox component doesn't work with newer versions of react-hook-form

1. Click on the Codesandbox link below which is taken from the official web-site of react-hook-form
2. Original link has 5.5.1 version of react-hook-form and the Checkbox component works well
3. On the left side, when you increase the version of react-hook-form dependency to for example 6.9.6 Checkbox component starts to not working.

**Codesandbox link**
https://codesandbox.io/s/react-hook-form-controller-079xx

**Comment by bluebill1049** [comment, marked as answer]
checkout v6 changelog: https://github.com/react-hook-form/react-hook-form/blob/master/CHANGELOG.md

https://react-hook-form.com/api#Controller

use render prop for Checkbox
<img width="943" alt="Screen Shot 2020-10-26 at 9 34 46 pm" src="https://user-images.githubusercontent.com/10513364/97162263-143ccf80-17d3-11eb-8210-6cfb099314f2.png">

here is an official CSB:
https://codesandbox.io/s/react-hook-form-v6-controller-qsd8r
